src/app/my25/[userId]/components/FavoriteMovieList.jsx

```diff
@@ -99,6 +98,10 @@ export function FavoriteMovieList({
 
   useEffect(() => keyCodeListener('Enter', () => setShowModal(true)), []);
 
+  const topTen = favorites.filter((m, i) => i < 10);
```

Comment:
Nit: have these be a constant. It's not likely, but if we do refactor and change the amounts it might be nice to just have one place to change it. ie:

```
const NUM_RATED = 10;
const NUM_HM = 15;

const COUNTED= NUM_RATED + NUM_HM;
const rated= favorites.filter((m, i) => i < NUM_RATED);
const honorableMentions = favorites.filter((m, i) => i >= NUM_RATED && i < COUNTED);
const overflow = favorites.filter((m, i) => i >= COUNTED)
```

Reply:
Yeah, good idea. I'm thinking we can put them in a separate constants file so that we can use them elsewhere (pop up message after adding a movie)